src/teststat/StatisticSum.cpp

```diff
@@ -0,0 +1,75 @@
+#include <StatisticSum.h>
+#include <ContainerTools.hpp>
+
+StatisticSum::StatisticSum(TestStatistic& s1_, TestStatistic& s2_){
+    AddStat(s1_);
+    AddStat(s2_);
+}
+
+double
+StatisticSum::Evaluate(){
+    double sum = 0;
+    for(size_t i = 0; i < fStats.size(); i++)
+        sum += fStats.at(i) -> Evaluate();
```

Comment:
Hey. Yea, that's what we want in this case - the log likelihoods sum. 
Could use likelihoods instead and support `*` but you usually run out of machine precision that way